src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```diff
@@ -33,8 +63,33 @@ $message-margin: $spacing + $close-icon-width + 5px;
       text-decoration: underline;
     }
   }
-}
 
-.announcement-message {
-  margin: 0 $message-margin;
+  &__container {
+    top: 66px;
```

Comment:
This should be a const.

Reply:
I believe this might be the navbar height from memory? If so, should be using that `const`

Reply:
yep, fixed in 2431677